### DISCUSSION TITLE
Problem with word break

### DISCUSSION THREAD

**Discussion by N6REJ** [community/community#29363] [New to GitHub] [answered]
How can I get the text for nodejs and php to stop wrapping under all the way to the left.
I want it to be 2 columns.
icons on the left , then text on the right.
https://github.com/Bearsampp/Bearsampp#readme

**Comment by airtower-luna** [comment, marked as answer]
Using a list you can't do that. Maybe a table is what you're looking for? See: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables

**Reply by N6REJ** [reply]
oh boy, now I feel dumb!!! ty!
